KNet/KNet.API/README.md

```diff
@@ -0,0 +1,11 @@
+# Producera och leverera mjukvara
+
+![Release CI/CD - Build, test and then run in azure](https://github.com/PGBSNH19/project-g2/workflows/Release%20CI/CD%20-%20Build,%20test%20and%20then%20run%20in%20azure/badge.svg)
+
+![CI - Build and test on development branch](https://github.com/PGBSNH19/project-g2/workflows/CI%20-%20Build%20and%20test%20on%20development%20branch/badge.svg)
+
+
+Link till project: <https://pgbsnh19.github.io/course-producera-leverera/assignments/project>
+
```

Comment:
Small suggestion: Add a guide on how to setup a local development environment and useful links to Website/API in production, jira documentation etc.